### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Peter-Dumbari**
In this PR:
-  I created a new branch feature
- I pull the code on my development branch
- I added some files and folders to set up my application.
- I ran linter check
- I created a pull request

### CODE REVIEW

File: src/components/Books/Books.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+// import './Books.css';
+
+function Books() {
+  return <div>Books</div>;
+}
+
```
Comment (brainconnect93) [inline src/components/Books/Books.jsx]: - [ ] Great effort working on the Bookstore, Initialize project with component milestone, but according to the requirement there are some thing you are currently missing. The requirement state that

1. A `<Books>` component, which contains the following components:
   - `<BookList>` which Renders a list of books using the `<Book>` component

2. `<Book>` which Takes the book data as a prop and renders the title and the author
   - Has a button for removing books: the button wouldn't remove the books yet, this is just a placeholder for now

So therefore kindly create some list of book using the <Book> component and also let it have a remove button but not necessary it is functioning at the moment.

File: src/components/CreateBookForm/Createbookform.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+// import './Createbookform.css';
+
+function Createbookform() {
+  return <div>Createbookform</div>;
+}
+
+export default Createbookform;
```
Comment (brainconnect93) [inline src/components/CreateBookForm/Createbookform.jsx]: - [ ] According to the requirement, I would kindly suggest you to work on this component as well so as to go inline with what we have in the project requirement

1. `<CreateBookForm>`
Should: 
* Renders a form with inputs for the title and the author of the book
* Has a button to submit
  * the button wouldn't add the book *yet*, this is just a placeholder for now

please kindly implement this in your project so as to have the proper output in your project.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Peter-Dumbari, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight👍

- Descriptive Pull Request title and summary. ✔️
- `react-router-dom` should be present in `package.json` ✔️
- `React Router` has been correctly set up in `<App>` ✔️
- Good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @Peter-Dumbari 
 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

  unnamed


## Highlights
- You implemented all suggested changes :heavy_check_mark: 
 
## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**